config/application.rb

```diff
@@ -17,6 +17,10 @@
 
 module BlogApp
   class Application < Rails::Application
+
+    # Background Job's
+    config.autoload_paths << Rails.root.join('app', 'jobs')
+
     # Settings in config/environments/* take precedence over those specified here.
```

Comment:
Thanks Ana... I really forgot about this "trick" to speed up the images loading process. ;)